packages/flutter_tools/test/general.shard/project_test.dart

```diff
@@ -738,6 +738,58 @@ apply plugin: 'kotlin-android'
         XcodeProjectInterpreter: () => xcodeProjectInterpreter,
         FlutterProjectFactory: () => flutterProjectFactory,
       });
+
+    testUsingContext('kotlin host app language with Gradle Kotlin DSL', () async {
+      final FlutterProject project = await someProject();
+
+        addAndroidGradleFile(project.directory,
+          gradleFileContent: () {
+            return '''
+plugins {
+    id "com.android.application"
+    id "kotlin-android"
+    id "dev.flutter.flutter-gradle-plugin"
+}
+''';
+        }, kotlinDsl: true);
```

Comment:
Same here bool above long strings.

Reply:
fixed in ae60874a573f5f677bd6f8c63ce6e34311d4b319